packages/fxa-auth-server/config/index.ts

```diff
@@ -1979,6 +1979,26 @@ const convictConf = convict({
       format: String,
     },
   },
+  graphql: {
+    contentfulApiKey: {
+      default: '',
+      doc: 'GraphQL Content API key for Contentful hCMS to fetch RP-provided content',
```

Comment:
Same suggestions here on the `doc` string as below to include Contentful doc URLs for these configs if possible (sorry I commented out of order).